docs/running-rules.md

```diff
@@ -9,39 +9,44 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in open-source Semgrep Registry that enables you to scan code without need to write anything custom.
+Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. Use Semgrep rules in the following ways:
 
-Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
+- Use existing [Semgrep Registry rules](#running-semgrep-registry-rules) or add new rules to Semgrep Registry and run them.
+- Local rules:
+  - Create and run [ephemeral rules](#ephemeral-rules) in the command line.
+  - Create and run [YAML-defined rules](#creating-and-using-yaml-defined-rules-file).
+- Run your [local rules simultaneously with Semgrep Registry rules](#running-multiple-rules-simultaneously). 
+
+You can run all rules on your local code or continuously in your Source Code Management (SCM) service (such as GitHub or GitLab) with Semgrep in CI.
 
+The following diagram displays the current number of Semgrep rules:
 <div className="lang-container" style={{marginBottom: '20px'}}>
   <iframe width="900" height="400" frameBorder="0" src="https://dashboard.semgrep.dev/metric/semgrep-rules.num/graph"></iframe>
```

Comment:
True true, it is not very clean with that black part... and might give too much information when folks want to try out rules. Maybe I can move his diagram to some appendix or: The interesting facts about rules section at the bottom of this document. Not sure how to create the counter. But it would be great here. @efortuna

Reply:
Moving at the bottom of the article to a new section.